tests/pytests/test_mro_drivers.py

```diff
@@ -42,8 +47,20 @@ def test_mro_load(test_kernels, label_type, kernel_type):
 
     assert compare_dicts(isd_obj, compare_isd) == []
 
-# ========= Test isislabel and isisspice driver =========
-class test_isis_isis(unittest.TestCase):
+@pytest.mark.parametrize("label_type, kernel_type", [('isis3', 'naif')])
+def test_mro_hirise_load(test_hirise_kernels, label_type, kernel_type):
+    label_file = get_image_label("PSP_001446_1790_BG12_0", label_type)
+
+    isd_str = ale.loads(label_file, props={'kernels': test_hirise_kernels})
+    compare_isd = get_isd('hirise')
+
+    isd_obj = json.loads(isd_str)
+    comparison = compare_dicts(isd_obj, compare_isd)
+    print(comparison)
```

Comment:
Did this print sneak in or do we want to keep it around?

Reply:
I want to keep it there for output on failure. Without it the failure in compare_dicts is difficult to parse